cmd/metal-api/internal/grpc/server.go

```diff
@@ -0,0 +1,81 @@
+package grpc
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	v1 "github.com/metal-stack/metal-api/pkg/api/v1"
+	"github.com/metal-stack/metal-lib/zapup"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/keepalive"
+	"io/ioutil"
+	"net"
+	"time"
+)
+
+func Serve(ws *WaitServer) {
+	logger := zapup.MustRootLogger().Sugar()
+
+	grpcPort := viper.GetInt("grpc-port")
```

Comment:
I think, we tried to prevent using viper from outside of the `main` package because it could get confusing when accessing variables from anywhere. Do you think it would be possible to wrap these parameters into a struct and pass this as a parameter?